src/Transactions/Transfer/TransferAddModalForm.js

```diff
@@ -21,7 +28,43 @@ import {
 
 const TRANSFER_FORM = 'transferForm';
 
-const TransferAddModalForm = ({ handleSubmit, onClose, funds }) => {
+const TransferAddModalForm = ({ budgetFund, handleSubmit, onClose, funds, store, dispatch, change }) => {
+  const formValues = getFormValues(TRANSFER_FORM)(store.getState());
+  const transferFrom = get(formValues, 'fromFundId');
+  const transferTo = get(formValues, 'toFundId');
+  const isTransferFromReqired = has(formValues, 'toFundId') ? transferTo !== budgetFund : false;
+  const validateTransferFrom = isTransferFromReqired ? { validate: validateRequired } : {};
+
+  const getOptionsFrom = () => (
```

Comment:
the same as above, what is the reason of defining a function here?

Reply:
fixed